components/infobox/wikis/rocketleague/infobox_league_custom.lua

```diff
@@ -163,6 +164,9 @@ function CustomLeague:defineCustomPageVariables(args)
 	Variables.varDefine('tournament_participant_number', 0)
 	Variables.varDefine('tournament_participants', '(')
 	Variables.varDefine('tournament_teamplayers', args.mode == _MODE_2v2 and 2 or 3)
+	Variables.varDefine('showh2h',
+		Logic.emptyOr(args.showh2h, tostring(tonumber(args.liquipediatier) < 5))
```

Comment:
- will error for invalid tier input (`nil < 5`)
- the `5` should be made a const imo (to align with code style)